db/schema.rb

```diff
@@ -19,9 +19,9 @@
   create_table "active_storage_attachments", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.string "name", null: false
     t.string "record_type", null: false
-    t.uuid "record_id"
     t.uuid "blob_id", null: false
     t.datetime "created_at", null: false
+    t.uuid "record_id"
```

Comment:
not sure this change is needed

Reply:
Oh this attribute just got reordered somehow after running migration, but nothing really changed

Reply:
can you remove that change from this PR?

Reply:
I mean I can manually revert it but not sure about benefits of it? Maybe I misunderstood you 🤔  ...